fe/src/index.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
```

Comment:
- remove this file below
![image](https://user-images.githubusercontent.com/73863685/218712818-fcaa9d9c-e12e-41b9-a373-fd3c39a35dee.png)